worlds/alttp/docs/multiworld_es.md

```diff
@@ -1,223 +1,112 @@
+   ROM.
+    1. Extrae la carpeta de tu emulador al Escritorio, o algún otro sitio que vayas a recordar.
+    2. Haz click derecho en un archivo ROM y selecciona **Abrir con...**
+    3. Marca la casilla junto a **Usar siempre este programa para abrir archivos .sfc**
+    4. Baja al final de la lista y haz click en el texto gris **Buscar otro programa en este PC**
+    5. Busca el archivo `.exe` de tu emulador y haz click en **Abrir**. Este archivo debería de encontrarse dentro de la carpeta que
+      extrajiste en el paso uno. 
 
 ### Obtener el fichero de parche y crea tu ROM
 
-Cuando te unes a una partida multiworld, debes proveer tu fichero YAML a quien sea el creador de la partida. Una vez
+Cuando te unas a una partida multiworld, se te pedirá enviarle tu archivo de configuración a quien quiera que esté creando. Una vez eso
 este hecho, el creador te devolverá un enlace para descargar el parche o un fichero zip conteniendo todos los ficheros
-de parche de la partida Tu fichero de parche debe tener la extensión `.aplttp`.
+de parche de la partida. Tu fichero de parche debe de tener la extensión `.aplttp`.
 
-Pon tu fichero de parche en el escritorio o en algún sitio conveniente, y haz doble click. Esto debería ejecutar
-automáticamente el cliente, y ademas creara la rom en el mismo directorio donde este el fichero de parche.
+Pon tu fichero de parche en el escritorio o en algún sitio conveniente, y hazle doble click. Esto debería ejecutar
+automáticamente el cliente, y además creará la rom en el mismo directorio donde este el fichero de parche.
 
 ### Conectar al cliente
 
 #### Con emulador
 
-Cuando el cliente se lance automáticamente, QUsb2Snes debería haberse ejecutado también. Si es la primera vez que lo
-ejecutas, puedes ser que el firewall de Windows te pregunte si le permites la comunicación.
+Cuando el cliente se lance automáticamente, SNI debería de ejecutarse en segundo plano. Si es la 
+primera vez que se ejecuta, tal vez se te pida permitir que se comunique a través del firewall de Windows
 
 ##### snes9x-rr
 
-1. Carga tu fichero de ROM, si no lo has hecho ya
+1. Carga tu fichero ROM, si no lo has hecho ya
 2. Abre el menu "File" y situa el raton en **Lua Scripting**
 3. Haz click en **New Lua Script Window...**
 4. En la nueva ventana, haz click en **Browse...**
-5. Navega hacia el directorio donde este situado snes9x-rr, entra en el directorio `lua`, y
-   escoge `multibridge.lua`
-6. Observa que se ha asignado un nombre al dispositivo, y el cliente muestra "SNES Device: Connected", con el mismo
-   nombre en la esquina superior izquierda.
+5. Selecciona el archivo lua conector incluido con tu cliente
+      - Busca en la carpeta de Archipelago `/SNI/lua/`.
+6. Si ves un error mientras carga el script que dice `socket.dll missing` o algo similar, ve a la carpeta de
+el lua que estas usando en tu gestor de archivos y copia el `socket.dll` a la raíz de tu instalación de snes9x.
 
```

Comment:
BNES-Plus is missing from the translation
This is a short section so I attempted to translate myself
```suggestion

##### BNES-Plus

1. Cargue su archivo ROM si aún no se ha cargado.
2. El emulador debería conectarse automáticamente mientras SNI se está ejecutando.

```